README.md

```diff
@@ -1,2 +1,45 @@
 # web-mapviewer
 The viewer application for maps and geodata
+
+## Project structure
+
+This is a Vue app, that is served through `src/main.js`.
+The app is divided into modules (or chunks) that are stored into `src/modules`. The goal is for each of these modules to be able to be externalized if needed (they should explicitly state their dependencies in their `index.js`)
+
+Each module should have a root component, called `{Name of the module}Module.vue` that loads all needed component into the template. It should also have a `README.md` file at the root explaining what this module is about.
+
+If the module needs to add data not related to something app wise (for instance, some internal state), a `store` folder can be created with an `index.js` file exporting a Vuex module.
+This module can then be imported in the store module (see below)
+
+### Store module
+ As there can be only one instance of a Vuex's store per app, the store module is there for that. It as the responsibility to instantiate Vuex, and add any module related state data to the store.
+See [its README.md](src/modules/store/README.md) for more details.
+
+## Project setup
+```
+npm install
```

Comment:
There's a number of deprecation warnings:
```
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated left-pad@1.3.0: use String.prototype.padStart()
```
I guess this is somewhat normal. But still the questions (since we start something new): 
- How are dependencies and their deprecations handled? 
- How risky is it to use these packages? (security, limits in browser support, ...)

Reply:
I've looked which packages brings those deprecated dependencies, and it's out of the scope of what we can change. Furthermore, most of them are from dev dependencies (vue dev tools), we can only hope for them to be upgraded and not include these outdated dependencies in the future

Reply:
regarding npm version, according to https://cli.vuejs.org/guide/installation.html it is needed to have Node.js version 8.9 or above (8.11.0+ recommended).

I can add a little paragraph about that in the README.md

Reply:
:+1:

Reply:
> I've looked which packages brings those deprecated dependencies, and it's out of the scope of what we can change. Furthermore, most of them are from dev dependencies (vue dev tools), we can only hope for them to be upgraded and not include these outdated dependencies in the future

Yeah, that JS-dependency jungle is sometimes a bit frightening...